### PR TITLE
Message content_type from Contracts

### PR DESCRIPTION
**Pull request by CDimonaco**

### CODE REVIEW

File: lib/wanda/messaging/adapters/amqp/publisher.ex [hunk 1]
```diff
@@ -16,7 +18,7 @@ defmodule Wanda.Messaging.Adapters.AMQP.Publisher do
   def publish_message(message, routing_key \\ "") do
     Logger.info("Publishing message #{inspect(message)}")
 
-    GenRMQ.Publisher.publish(__MODULE__, message, routing_key)
+    GenRMQ.Publisher.publish(__MODULE__, message, routing_key, [{:content_type, Contracts.content_type}])
```
Comment (arbulu89) [inline lib/wanda/messaging/adapters/amqp/publisher.ex:21]: `Contracts.content_type` is a function, right? Don't you need to finish with `()`?
I think at least `Credo` should complain about

Comment (CDimonaco) [reply lib/wanda/messaging/adapters/amqp/publisher.ex:21]: You're right, the rubyst in me take over everything 🤣

Comment (CDimonaco) [reply lib/wanda/messaging/adapters/amqp/publisher.ex]: fixed, make ruby great again! 🤣

**Review by arbulu89 — APPROVED** [review]

**Review by dottorblaster — APPROVED** [review]
LGTM

**Pull request merged**